chia/simulator/setup_services.py

```diff
@@ -39,6 +40,17 @@
 log = logging.getLogger(__name__)
 
 
+@contextmanager
+def create_lock_and_load_config(certs_path: Path, root_path: Path) -> Iterator[Dict[str, Any]]:
+    init(None, root_path)
+    init(certs_path, root_path)
+    path = config_path_for_filename(root_path=root_path, filename="config.yaml")
+    # Using localhost leads to flakiness on CI
```

Comment:
that's a pretty sparse comment. I assume because some machines resolve it to IPv6, while still not supporting IPv6 or not binding our listen socket to both IPv4 and IPv6.